src/utils/legacy/route.ts

```diff
@@ -2010,61 +2014,71 @@ LegacyAPI.post("/GetSurveys", [_authorize], async (req: Request, res: Response)
       ErrorMessage: "Specify valid User Id.",
     } as APIResponse)
   }
-  const resultQuery2 = await SQL!.request().query(`
-        SELECT 
-            SurveyID, 
-            SurveyName, 
-            Instructions AS Instruction, 
-            ISNULL(Language, 'en') AS LanguageCode, 
-            IsDeleted, 
-            (
-                SELECT 
-                    QuestionID AS QuestionId,
-                    QuestionText,
-                    (CASE 
-                        WHEN AnswerType = 1 THEN 'LikertResponse'
-                        WHEN AnswerType = 2 THEN 'ScrollWheels'
-                        WHEN AnswerType = 3 THEN 'YesNO'
-                        WHEN AnswerType = 4 THEN 'Clock'
-                        WHEN AnswerType = 5 THEN 'Years'
-                        WHEN AnswerType = 6 THEN 'Months'
-                        WHEN AnswerType = 7 THEN 'Days'
-                        WHEN AnswerType = 8 THEN 'Textbox' 
-                    END) AS AnswerType,
-                    IsDeleted,
-                    (
-                        SELECT 
-                            OptionText
-                        FROM SurveyQuestionOptions
-                        WHERE SurveyQuestions.QuestionID = SurveyQuestionOptions.QuestionID
-                        FOR JSON AUTO, INCLUDE_NULL_VALUES
-                    ) AS QuestionOptions,
-                    CAST(0 AS BIT) AS EnableCustomPopup,
-                    Threshold AS ThresholdId,
-                    Operator AS OperatorId,
-                    Message AS CustomPopupMessage
-                FROM SurveyQuestions
-                WHERE Survey.SurveyID = SurveyQuestions.SurveyID
-                FOR JSON AUTO, INCLUDE_NULL_VALUES
-            ) AS Questions
-        FROM Survey 
-        WHERE IsDeleted = 0
-        AND AdminID IN (
-            SELECT 
-                AdminID 
-            FROM Users 
-            WHERE IsDeleted = 0 
-                AND UserID = ${UserID}
-        )
-        FOR JSON AUTO, INCLUDE_NULL_VALUES
-    ;`)
 
-  return res.status(200).json({
-    ErrorCode: 0,
-    ErrorMessage: "Get surveys detail.",
-    Survey: resultQuery2.recordset.length > 0 ? resultQuery2.recordset[0] : [],
-    LastUpdatedDate: new Date().toISOString().replace(/T/, " ").replace(/\..+/, ""),
-  } as APIResponse)
+  let userData = (req as any).AuthUser
+  let decryptedUser = Decrypt(userData.Email) + ":" + Decrypt(userData.Password, "AES256")
+  let encryptUserData = Buffer.from(decryptedUser).toString("base64")
+  let surveyArray: any = []
+  ;(async () => {
+    const response = await fetch(app_url + "/activity", {
+      method: "GET",
+      headers: { Authorization: "Basic " + encryptUserData },
+    })
```

Comment:
We should NOT be making an HTTP call to local code. Please just call `ActivityRepository._select` directly.